packages/forwarded/src/index.ts

```diff
@@ -41,7 +28,20 @@ export function parse(header: string): string[] {
   }
 
   // final address
-  if (start !== end) list.push(header.substring(start, end))
 
+  if (start !== end) list.push(header.substring(start, end))
```

Comment:
why c8 ignore here?

btw let's not add ignored coverage, so we don't miss anything

Reply:
it's kinda buggy so it thinks the bracket is uncovered. 
![Screenshot 2023-08-03 at 14-06-22 Code coverage report for packages_forwarded_src_index ts](https://github.com/tinyhttp/tinyhttp/assets/43901677/98198510-5620-49a0-a39a-7fff8a9aa262)

Reply:
it's fine, it's probably something else causing the cov miss

Reply:
dunno
https://github.com/bcoe/c8/issues/81